test/question.spec.js

```
@@ -65,6 +75,20 @@ describe('Question Api Exists', () => {
         .patch('/api/v1/questions/0/upvote')
         .end((err, res) => {
           expect(res.statusCode).to.equal(200);
+          expect(res.body).to.be.an('object');
+          expect(res.body).to.have.property('data');
```

Same thing here, why are you only checking for the property data?

---

I've been having issues working my way around checking for the keys in my data object. I'm thinking the error is because I am using a dummy database because when I use `expect(res.body).to.have.keys({ 'title', 'body' })`, my test kept failing saying it's expecting an additional key '0'. So I decided to only check for the variable holding the object inside.